README.md

```diff
@@ -53,6 +53,14 @@ Usage: relation-graph [options]
         Set log level to INFO
 ```
 
+## Installation
+### MacOS
```

Comment:
I don't really install things on Linux this way, but I imagine it's similar. Not sure how to do installation of this sort on Windows, though.